app/models/request.rb

```diff
@@ -27,4 +27,8 @@ class Request < ApplicationRecord
   scope :decision, ->(decision) { where decision: decision }
   scope :state, ->(state) { where state: state }
   scope :requester, ->(requester) { where requester: requester }
+
+  def as_json(options={})
+    super(:include => [ :workflow, :stages ])
```

Comment:
In general we don't need to expose `belong_to` relation. Common practice is to provide a `links` sections in the payload; else we can simply include the `workflow_id`. 
Because stages are so tightly integrated with request we can treat stages as embedded although they are stored in another table, we can do `super(:include => [:stages])`

Reply:
It's no way to provide such a link through swagger spec. I will simply include workflow_id in response. Similar logic will apply to other models: just include the associated ids.